config/Config-template.groovy

```diff
@@ -155,6 +155,9 @@ environments { development {
     com.recomdata.heartbeatLaps = 900
 } }
 
+// Maximum concurrent sessions for a user (-1: unlimited)
+// com.recomdata.maxConcurrentUserSessions = 10
```

Comment:
Thanks @forus for catching this. Probably not a bad Idea, also, to make a JIRA request for the others.